src/js/BookReader/options.js

```diff
@@ -207,26 +207,59 @@ export const DEFAULT_OPTIONS = {
   getEmbedCode: null,
 
   controls: {
+    bookLeft: {
+      visible: true,
+      label: 'Flip left',
+      className: 'book_left',
+      iconClassName: 'left-arrow'
+    },
+    bookRight: {
+      visible: true,
+      label: 'Flip right',
+      className: 'book_right',
+      iconClassName: 'left-arrow hflip'
+    },
     onePage: {
-      label: 'One-page view',
       visible: true,
+      label: 'One-page view',
       className: 'onepg',
+      iconClassName: 'onepg'
     },
     twoPage: {
-      label: 'Two-page view',
       visible: true,
+      label: 'Two-page view',
       className: 'twopg',
+      iconClassName: 'twopg'
     },
     thumbnail: {
-      label: 'Thumbnail view',
       visible: true,
+      label: 'Thumbnail view',
       className: 'thumb',
+      iconClassName: 'thumb'
     },
     viewmode: {
-      visible: false,
+      visible: true,
```

Comment:
does `view mode` need a `label` field, too?

Reply:
No, it will automatically added later when BR is initialised and it know on what page view you are currently.